app/renderer/actions.js

```diff
@@ -1325,6 +1325,13 @@ const useEnteredData = () => {
     }
 }
 
+const useCurrentUrl = () => getPageUrl()
+
+const useCurrentOrigin = () => {
```

Comment:
This doesn't feel like the right place to have these templates stored, since they are now also treated as actions. The command replacement logic is good, but this needs moving, probably to `command.js`, also means less imports.